File: dataset_reduzido/avatar/Python/Code/codeforces_624_B.py
import sys
input = sys.stdin.readline
n = int ( input ( ) )
a = list ( map ( int , input ( ).split ( ) ) )
pre = float ( 'inf' )
ans = 0
a.sort ( reverse = True )
for j in range ( n ) :
    ans += max ( 0 , min ( pre - 1 , a [ j ] ) )
    pre = max ( 0 , min ( pre - 1 , a [ j ] ) )
print ( ans )

File: output/Qwen_repaired_1/avatar/Python/Java/codeforces_624_B.java
import java.util.*;
class Main{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        int n=Integer.parseInt(sc.nextLine());
        String[] arr=sc.nextLine().split(" ");
        List<Integer> a=Arrays.stream(arr).map(Integer::parseInt).collect(Collectors.toList());
        Collections.sort(a,Collections.reverseOrder());
        long ans=0;
        double pre=Double.MAX_VALUE;
        for(int j:a){
            ans+=Math.max(0,Math.min((int)pre-1,j));
            pre=Math.max(0,Math.min((int)pre-1,j));
        }
        System.out.println(ans);
        sc.close();
    }
}